src/resources/Microsoft.Authorization/policySetDefinitions/Enforce-EncryptTransit_20241211.json

```diff
@@ -0,0 +1,956 @@
+{
+    "name": "Enforce-EncryptTransit_20241211",
+    "type": "Microsoft.Authorization/policySetDefinitions",
+    "apiVersion": "2021-06-01",
```

Comment:
does this need bumping to a version that supports versioning? And if so, we probably need to bump all the initiatives for this, even though this property isnt used by this RI, it may well be by the library and other RIs so lets make this accurate 👍

Reply:
Done